samples/gcc/.mainframer/remoteignore

```diff
@@ -0,0 +1,2 @@
+# Usually you don't need to sync sources back from remote to local machine.
+*.c
```

Comment:
Please take a look at https://github.com/github/gitignore/blob/master/C.gitignore, maybe it makes sense to include something from it.